test/evergreen.yml

```diff
@@ -4568,6 +4568,14 @@ tasks:
                 cat $t && rm $t
             fi
 
+            # Code complexity statistics
+
+            python3 "../metrixplusplus/metrix++.py" view --format=python > code_complexity_summary.txt
+            python3 "../metrixplusplus/metrix++.py" export --db-file=metrixpp.db > metrixpp.csv
+
+            cd ../test/evergreen
```

Comment:
Is it necessary to change directory here?

Reply:
Fixed in 7cc186a02197e388c9a2fcaa97331d24c84376a5 to use the relative path